sentences/pt/cover_HassTurnOn.yaml

```diff
@@ -10,11 +10,25 @@ intents:
           domain: cover
 
       - sentences:
-          - <abre>  [as|os|a|o] [(cortina|cortinas|cortinado|cortinados|estore|estores|persiana|persianas)] <na_zona>
+          - <abre>  [as|os|a|o] <estores> <na_zona>
         response: cover
         slots:
           device_class:
             - blind
             - curtain
             - shutter
           domain: cover
+
+      # abrir estores na mesma divisão que um dispositivo de voz
+      - sentences:
+          - <abre>  [as|os|a|o] <estores> [aqui]
```

Comment:
```suggestion
          - <abre> [as|os|a|o] <estores> [aqui]
```

Reply:
Maybe we can move `[as|os|a|o]` to the `estores` slot. I think it is always used as a  combination

Reply:
I confirmed that it is used always together with the optionals so i followed your suggestion and changed as you said. Thanks ;)